VelodyneHDL/IO/Lidar/Velodyne/vtkVelodynePacketInterpreter.cxx

```diff
@@ -971,7 +973,7 @@ void vtkVelodynePacketInterpreter::PushFiringData(unsigned char laserId, unsigne
   double distanceM;
   double pos[3];
   bool applyIntensityCorrection =
-    this->WantIntensityCorrection && this->IsHDL64Data && !(this->SensorPowerMode == CorrectionOn);
+    this->WantIntensityCorrection && this->IsHDL64Data; // TODO not sure what this gets set or what it means && !(this->SensorPowerMode == CorrectionOn);
```

Comment:
Hey @bcasselsND, since you asked about merging upstream (which I do want to do), this is an example of a Vinay change that I kept, which would probably require a review between VeloView and us...which is ok?

Reply:
Sure. Just call it out in the upstream pull request. They can decide to reject those changes. If they do, we'll figure out if we need those changes.

Reply:
This feels like one of our hacks... we should clean this up before merging upstream.